src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_help.py

```diff
@@ -10,4 +10,12 @@
 helps['configure'] = """
             type: command
             short-summary: Configure Azure CLI 2.0 Preview or view your configuration. The command is interactive, so just type `az configure` and respond to the prompts.
+            examples:
+                - name: configure a default resource group
+                  text: >
+                    az configure --setting resource_group myRG
```

Comment:
Agreed. Like i said, I am thinking we just use a dedicated section `[default]`, and use `group`